backend/app/donation/routes.py

```diff
@@ -3,14 +3,12 @@
 from . import donation
 
 
-@donation.route("", methods=["GET"])
-def get_donation_amounts_by_email():
+@donation.route("/amount", methods=["GET"])
+def get_donation_amount():
     donations = Donation.query.all()
 
     email = request.args.get("email")
     if email is not None:
-        if email == "":
-            abort(404, "Invalid email address")
 
         filtered_donation = list(
             filter(lambda donations: (donations.email == email), donations)
```

Comment:
@peterghrong is it possible to change from `donations` to `donation` within the lambda function? As it is one donation only rather than the entire list of donations.

Reply:
Thank you for pointing this out Curtis, I have changed it accordingly